src/__tests__/classifiedConfig.test.js

```diff
@@ -0,0 +1,38 @@
+import { getInjector } from '../injector';
```

Comment:
We need a separate test file as the creation of the nodule needs to be done asynchronously (in the beforeEach). If we do this in the individual test, the fields will not be properly bound.